### ISSUE TITLE
TestJWSWithCertificateChain failed with go1.15

### ISSUE THREAD

**Issue by zhsj** [square/go-jose#330]
```
=== RUN   TestJWSWithCertificateChain
    jws_test.go:615: failed to verify certificate chain for test case 0: x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
--- FAIL: TestJWSWithCertificateChain (0.00s)
```

**Comment by pippo** [comment]
Should this be closed, as #332, addressing the issue, has been merged?

**Comment by csstaub** [comment]
Yes, though I'll also have to tag a new release.
